src/status_im/subs/activity_center.cljs

```diff
@@ -9,6 +9,13 @@
  (fn [activity-center]
    (:notifications activity-center)))
 
+(re-frame/reg-sub
+ :activity-center/supported-notifications
+ :<- [:activity-center/notifications]
+ (fn [notifications]
+   (->> notifications
+        (filter #(types/all-supported (:type %))))))
```

Comment:
Minor detail: the thread macro is unnecessary with just one transformation.

Reply:
It tends to be seen as superfluous in my experience from other Clojure repos. In the CSG https://guide.clojure.style/#threading-macros there's a recommendation to use it to reduce heavy nesting. I've seen most Clojure code tends to use the thread macros when there are two or more transformations (but it's not a rule).

If you add a line to separate the transformation it can look clean.

```clojure
(filter #(types/all-supported (:type %))
        notifications)
```

Or if you create a predicate it looks cleaner:

```clojure
(filter supported-notification? notifications)
```

Reply:
Thank you, @ilmotta, for the explanation. I think the last approach is the best one, and I’ve updated the code accordingly.